packages/react-composites/src/composites/CallComposite/pages/NoticePage.tsx

```diff
@@ -24,6 +30,42 @@ export interface NoticePageProps {
   dataUiId: string;
   disableStartCallButton?: boolean;
   pageStyle?: IStyle;
+  /* @conditional-compile-remove(end-of-call-survey) */
+  /**
+   * Options for end of call survey
+   */
+  surveyOptions?: {
+    /**
+     * Hide call survey at the end of a call.
+     * @defaultValue true
```

Comment:
is the default value `true`? (that we hide by default?)
If so this should be `showSurvey` and default value false

Reply:
I like the idea that we should be showing by default as we have been with other features. To make this work do they need to provide information that would make this a case for hiding by default? I wonder if we can conditionally render the survey if they provide the callback needed to submit the survey and remove this API all together?